Products/CMFPlone/browser/okay.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from Products.Five.browser import BrowserView
+
+
+class OK(BrowserView):
+    """Returns OK.
+
+    Useful for automated checks, for example memmon, to see if the site
+    is still available.  For this you don't want to query a possibly
+    expensive page.  And you don't want to query anything that may be
+    stored in an intermediate caching server.
+
+    Calling /ZopeTime can work too.  But if you use
+    experimental.publishtraverse, this will either give you a NotFound,
+    or log a warning each time it is called, due to lacking
+    permissions.  And this may happen in core Plone in the future too.
+    """
+
+    def __call__(self):
+        # Make really sure this response is not cached.
+        set_header = self.request.response.setHeader
+        set_header('Expires', 'Wed, 11 Jan 2017 22:00:00 GMT')
+        set_header('Cache-Control', 'max-age=0, must-revalidate, private')
```

Comment:
I took the value that [plone.app.caching](https://github.com/plone/plone.app.caching/blob/d8c5ae4115a4b06872a3d7092141c540338b398e/plone/app/caching/operations/utils.py#L106-L119) sets. I first had a header similar to what you suggest, but according to `plone.app.caching` it can trip up Internet Explorer in some cases.
Not that this is meant to be used by IE or any other browser, but it seems best to stick to what we already do. I will add a note.